etcdmain/etcd.go

```diff
@@ -58,10 +58,14 @@ func startEtcdOrProxyV2() {
 	err := cfg.parse(os.Args[1:])
 	lg := cfg.ec.GetLogger()
 	if err != nil {
-		lg.Warn("failed to verify flags", zap.Error(err))
+		if lg != nil {
```

Comment:
Is it better to add some output of invalid parameter prior to exit?

Reply:
Using fmt.Println ?

Reply:
I have noticed this issue is introduced with PR https://github.com/etcd-io/etcd/commit/0a05f1a1312d45503c06b1e8795466616a91545f. Will the package `github.com/coreos/pkg/capnslog` be removed permanently?